styles.css

```diff
@@ -0,0 +1,294 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-color: #ff6969;
+}
+
+.header-toolbar {
+  display: flex;
+  flex-direction: row;
+  min-height: 4rem;
+  align-items: center;
+  padding: 15px;
+  color: white;
+  opacity: 1;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.my-logo {
+  color: #fff;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  font-family: Roboto, sans-serif;
+  letter-spacing: 0.03em;
+  text-decoration: none;
+}
+
+.my-logo:hover {
```

Comment:
You  should  group all styles related that have same properties style because to be easier maintain and understand.